src/ripple/protocol/impl/STAmount.cpp

```diff
@@ -1092,28 +1062,28 @@ bool
 operator< (STAmount const& lhs, std::uint64_t rhs)
 {
     // VFALCO Why the cast?
-    return lhs.getSNValue() < static_cast <std::int64_t> (rhs);
+    return getSNValue(lhs) < static_cast <std::int64_t> (rhs);
```

Comment:
Gosh, I hate to spoil this perfect run of "nothing to say" by quibbling that the code generally has a space after function names if there's a parameter.